public/css/filter.css

```
@@ -0,0 +1,100 @@
+.filter {
+    display: flex;
```

```suggestion
  display: flex;
```
CSS should be indented with 2 spaces instead of 4 spaces